### PR TITLE
Spanish Strings 2

### PR DESCRIPTION
**Pull request by Shield1739**
"Fixed" most Spanish strings and added one that was missing.

(now this is the correct one)

### CODE REVIEW

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -669,7 +669,7 @@
 		<Key ID="STR_ANOTF_MEnabled">
 			<Original>Player Markers Enabled.</Original>
 			<Czech>Hráč Markers Povoleno.</Czech>
-			<Spanish></Spanish>
+			<Spanish>Marcadores de Jugadores Abilitados</Spanish>
```
Comment (xjose93) [inline Altis_Life.Altis/stringtable.xml]: Abilitados => Habilitados.

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -771,7 +771,7 @@
 			<Original>You have no business using this.</Original>
 			<Czech>Nemáš podnikání pomocí tohoto.</Czech>
 			<French>Vous avez pas d'affaires en utilisant cela.</French>
-			<Spanish>No tiene sentido utilizar este.</Spanish>
+			<Spanish>No tienes permiso de usar esto.</Spanish>
```
Comment (xjose93) [inline Altis_Life.Altis/stringtable.xml]: No tienes permiso para usar esto.

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -782,7 +782,7 @@
 			<Original>Admin %1 has opened the debug console.</Original>
 			<Czech>Admin %1 otevřel ladění konzoli.</Czech>
 			<French>Administrateur %1 a ouvert la console de débogage.</French>
-			<Spanish>Administrador %1 ha abierto la consola de depuración.</Spanish>
+			<Spanish>Administrador %1 ha abierto la consola de debug.</Spanish>
```
Comment (xjose93) [inline Altis_Life.Altis/stringtable.xml]: Depuracion its ok in spanish.

File: Altis_Life.Altis/stringtable.xml [hunk 4]
```diff
@@ -5543,7 +5543,7 @@
 			<Original>You have repaired your vehicle for $500.</Original>
 			<Czech>Máte opravit vůz za 500 $.</Czech>
 			<French>Vous avez réparer votre véhicule 500 $.</French>
-			<Spanish>Hay que reparar su vehículo por $ 500.</Spanish>
+			<Spanish>Has repararado tu vehículo por $ 500.</Spanish>
```
Comment (xjose93) [inline Altis_Life.Altis/stringtable.xml]: Has reparado tu vehículo por $500.

File: Altis_Life.Altis/stringtable.xml [hunk 5]
```diff
@@ -5427,7 +5427,7 @@
 			<Original>This house is already owned even though you shouldn't be seeing this hint...</Original>
 			<Czech>Tento dům je již ve vlastnictví i když by nemělo být vidět tuto radu ...</Czech>
 			<French>Cette maison est déjà propriétaire, même si vous ne devriez pas voir cette astuce ...</French>
-			<Spanish>Esta casa es de propiedad ya pesar de que no debería estar viendo esta pista ...</Spanish>
+			<Spanish>Esta casa ya tiene dueño, aunque no debería estar viendo esta pista ...</Spanish>
```
Comment (xjose93) [inline Altis_Life.Altis/stringtable.xml]: Esta casa ya tiene dueño, aunque no debería estar viendo este mensaje...

File: Altis_Life.Altis/stringtable.xml [hunk 6]
```diff
@@ -4768,7 +4768,7 @@
 			<Original>You must select a perp.</Original>
 			<Czech>Je třeba vybrat pachatele.</Czech>
 			<French>Vous devez sélectionner un perp.</French>
-			<Spanish>Debe seleccionar un asesino.</Spanish>
+			<Spanish>Debe seleccionar un perpetruador.</Spanish>
```
Comment (xjose93) [inline Altis_Life.Altis/stringtable.xml]: "asesino" or "criminal" its ok

**Comment by Shield1739** [comment]
Abilitados => Habilitados.
No tienes permiso para usar esto.

yup u're right about these two. And "Criminal" prob does a better fit. When I get home I'll change those two.

**Pull request closed without merging**